test/zippy/mzcompose.py

```diff
@@ -49,6 +51,22 @@ def __str__(self) -> str:
         return self.value
 
 
+def parse_timedelta(arg: str) -> timedelta:
```

Comment:
I'm not so happy with this logic, seems a bit complex, but timedelta doesn't seem to have a parse for this.

Reply:
Yes, I searched for a way to do this using the modules in requirements.txt but could not find any.

Reply:
e.g. we already import humanize, but it can not be used to parse its own humanized output.